package.json

```diff
@@ -132,19 +132,18 @@
     "postcss": "^8.3.5",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "rollup": "^2.53.1",
+    "rollup": "^4.1.4",
     "rollup-plugin-analyzer": "^4.0.0",
-    "rollup-plugin-copy": "^3.4.0",
-    "rollup-plugin-polyfill-node": "^0.10.1",
-    "rollup-plugin-postcss": "^4.0.0",
-    "rollup-plugin-scss": "^3.0.0",
-    "rollup-plugin-visualizer": "^5.9.0",
+    "rollup-plugin-copy": "^3.5.0",
+    "rollup-plugin-polyfill-node": "^0.12.0",
+    "rollup-plugin-postcss": "^4.0.2",
+    "rollup-plugin-scss": "^4.0.0",
+    "rollup-plugin-typescript2": "^0.36.0",
+    "rollup-plugin-visualizer": "^5.9.2",
```

Comment:
Nice but I hope nothing breaks after all these change.

Reply:
I don't see any significant changes that stand out!